Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/CoolingTowerParellel.mo

```diff
@@ -0,0 +1,296 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems;
+model CoolingTowerParellel
+  "Multiple identical cooling towers in parallel connection"
+  extends Buildings.Applications.DataCenters.ChillerCooled.Equipment.BaseClasses.SignalFilter(
+    riseTimeValve=30,
+    use_inputFilter=true,
+    final numFil=num);
+  parameter Integer num(
+    final min=1)=2
+    "Number of cooling towers";
+  replaceable package Medium=Buildings.Media.Water
+    "Condenser water medium";
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial
+    "Type of energy balance: dynamic (3 initialization options) or steady state"
+    annotation (Evaluate=true,Dialog(tab="Dynamics",group="Equations"));
+  parameter Boolean show_T=true
+    "= true, if actual temperature at port is computed"
+    annotation (Dialog(tab="Advanced",group="Diagnostics"));
+  parameter Modelica.SIunits.MassFlowRate m_flow_nominal
+    "Nominal mass flow rate of condenser water in each tower"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.Pressure dp_nominal
+    "Nominal pressure difference of the tower"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Real ratWatAir_nominal(
+    final min=0,
+    final unit="1")=0.625
+    "Design water-to-air ratio"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.Temperature TAirInWB_nominal
+    "Nominal outdoor (air inlet) wetbulb temperature"
+    annotation (Dialog(group="Heat transfer"));
+  parameter Modelica.SIunits.Temperature TWatIn_nominal
+    "Nominal water inlet temperature"
+    annotation (Dialog(group="Heat transfer"));
+  parameter Modelica.SIunits.TemperatureDifference dT_nominal
+    "Temperature difference between inlet and outlet of the tower"
+    annotation (Dialog(group="Heat transfer"));
+  parameter Modelica.SIunits.Power PFan_nominal
+    "Fan power"
+    annotation (Dialog(group="Fan"));
+  Medium.ThermodynamicState sta_a=Medium.setState_phX(
```

Comment:
Should these thermodynamic states be protected? I don't believe they should need to be accessed externally.